docs/core-concepts/core-sdk/cli/using-ci-with-cli.md

```diff
@@ -0,0 +1,41 @@
+# Using .NET Core CLI tools in Continuous Integration
```

Comment:
Nope, these are tools. Don't forget, we have a different installer for the shared fx, you could just use that...